src/_pytest/fixtures.py

```diff
@@ -54,10 +56,10 @@ def pytest_sessionstart(session):
     session._fixturemanager = FixtureManager(session)
 
 
-scopename2class = {}
+scopename2class: Dict[str, type] = {}
```

Comment:
this should be `Dict[str, Type[baseclass]` where `baseclass` is a common ancestor of the types used here -- (iirc it's `Node`? I'd have to look it up can't tell directly from the diff :D)